src/index.js

```diff
@@ -24,6 +24,7 @@ import api from './api/liskApi';
 import crypto from './crypto';
 import * as time from './transactions/utils/time';
 import transaction from './transactions';
+import * as utils from './transactions/utils';
```

Comment:
Prefer exporting it within the transactions module for access via `require('lisk-js').transactions.utils`, otherwise we'll have to refactor when it comes to switching to a monorepo.